src/modules/common/components/bars/topBar/networkName.js

```diff
@@ -1,60 +1,35 @@
 /* eslint-disable max-statements */
-import React, { useMemo, useState } from 'react';
+import React from 'react';
 import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
-import Dropdown from 'src/theme/Dropdown/dropdown';
-import Icon from 'src/theme/Icon';
-import networks, { networkKeys } from '@network/configuration/networks';
+import Tooltip from 'src/theme/Tooltip';
 import useSettings from '@settings/hooks/useSettings';
 import styles from './network.css';
 
 const Network = () => {
-  const [showDropdown, setShowDropdown] = useState(false);
   const { status } = useSelector((state) => state.network);
-  const { mainChainNetwork, setValue } = useSettings('mainChainNetwork');
+  const { mainChainNetwork } = useSettings('mainChainNetwork');
   const { t } = useTranslation();
 
   const activeNetworkName = mainChainNetwork?.name;
   const statusColor = status.online ? styles.online : styles.offline;
 
-  const closeDropdown = () => setShowDropdown(false);
-
-  const setActiveNetwork = (name) => {
-    setValue(networks[name]);
-    closeDropdown();
-  };
-
-  const activeNetworkIndex = useMemo(
-    () => Object.values(networkKeys).indexOf(activeNetworkName),
-    [activeNetworkName]
-  );
-
   return (
     <>
-      <section className={styles.wrapper} onClick={() => setShowDropdown(true)}>
+      <section className={styles.wrapper}>
         <span className={`${styles.status} ${statusColor}`} />
         <div className={styles.message}>
-          <span className="network-name">{t(activeNetworkName)}</span>
-          <Icon name="dropdownArrowIcon" />
+          <Tooltip
+            indent
+            tooltipClassName={styles.tooltipContainer}
+            position="bottom left"
+            size="maxContent"
+            content={<span className="network-name">{t(activeNetworkName)}</span>}
+          >
+            <span>{mainChainNetwork?.serviceUrl}</span>
+          </Tooltip>
         </div>
       </section>
-      <Dropdown
-        showDropdown={showDropdown}
-        active={activeNetworkIndex}
-        className={styles.dropdown}
```

Comment:
Should also clean up the  css file from unused rules: `dropdown`, `networkItem`, `title`